packages/playwright/src/makeTest.ts

```diff
@@ -14,6 +14,7 @@ import {
   trackRun,
   DEFAULT_GLOBAL_RESOURCE_ARCHIVE_TIMEOUT_MS,
 } from '@chromatic-com/shared-e2e';
+import { join } from 'node:path';
```

Comment:
Nitpick -- looks like this is imported [differently](https://github.com/chromaui/chromatic-e2e/blob/main/packages/shared/src/write-archive/index.ts#L1) in `write-archive`. Can these be unified?

Reply:
Done!